server/DBConfig.js

```diff
@@ -0,0 +1,8 @@
+const { Pool } = require("pg");
+const videosPool = new Pool({
+  connectionString: "postgres://full_stack_database_jp62_user:JfOA8mzl7maxtJEnTL28nmslJ5JSqVfe@dpg-ck2p2hvqj8ts73fm2m5g-a.oregon-postgres.render.com/full_stack_database_jp62",
```

Comment:
❗ You've just made your database connection string public! You need to store strings like this in env variables 👀

Reply:
I have tried before to use variables to store it but I got an error then I used this way